apps/web/src/containers/NoteView/staked-note/staked-note.tsx

```diff
@@ -140,15 +151,15 @@ export const StakedNote = () => {
               size="large"
               sx={{
                 fontFamily: 'Avenir Next',
```

Comment:
You can revert this disabling of the stake note thing, I will merge a PR after you to get this to work. It will sit at the `/stake` route.